.github/workflows/ci-unit-tests-docker.yml

```diff
@@ -21,6 +22,16 @@ on:
         default: false
 
   pull_request:
+    # Skip PRs from external repositories, let them pass, and then Mergify will check them
+    branches:
+      # Include all branches
+      - '**'
+      # Exclude external repositories
+      - '!**/**'
+      # But include mergify and dependabot
+      - 'mergify/**'
+      - 'dependabot/**'
```

Comment:
This is wrong and needs to be turned into a job condition (probably just the build job, because it's already conditional) and a bunch of dependencies.